source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -52,16 +52,33 @@ def __init__(self, instance, service_url, id='geovelo', timeout=10, api_key=None
         self.breaker = pybreaker.CircuitBreaker(fail_max=app.config['CIRCUIT_BREAKER_MAX_GEOVELO_FAIL'],
                                                 reset_timeout=app.config['CIRCUIT_BREAKER_GEOVELO_TIMEOUT_S'])
 
-    @staticmethod
-    def _pt_object_summary(pt_object):
+    @classmethod
+    def _pt_object_summary_isochrone(cls, pt_object):
         coord = get_pt_object_coord(pt_object)
         return [coord.lat, coord.lon, getattr(pt_object, 'uri', None)]
 
-    @staticmethod
-    def _make_request_arguments(origins, destinations):
+    @classmethod
+    def _make_request_arguments_isochrone(cls, origins, destinations):
         return {
-            'starts': [Geovelo._pt_object_summary(o) for o in origins],
-            'ends': [Geovelo._pt_object_summary(o) for o in destinations],
+            'starts': [cls._pt_object_summary_isochrone(o) for o in origins],
+            'ends': [cls._pt_object_summary_isochrone(o) for o in destinations],
+        }
+
+    @classmethod
+    def _make_request_arguments_direct_path(cls, origin, destination):
+        coord_orig = get_pt_object_coord(origin)
+        coord_dest = get_pt_object_coord(destination)
+        return {
+            'waypoints': [
+                {'latitude': coord_orig.lat, 'longitude': coord_orig.lon},
+                {'latitude': coord_dest.lat, 'longitude': coord_dest.lon}
+            ],
+            'bikeDetails': {
+                'profile': 'BEGINNER',
```

Comment:
I add a comment on possible values.
It seems too soon to put that in the connector or at call (following profile and bike speed?). We have to wait until geovelo manages parameters consistently in isochrone and in direct_path.